dist/images/ovnkube.sh

```diff
@@ -852,9 +852,12 @@ ovn-node() {
   fi
 
   OVN_ENCAP_IP=""
-  ovn_encap_ip=$(ovs-vsctl --if-exists get Open_vSwitch . external_ids:ovn-encap-ip | tr -d '\"')
-  if [[ $? == 0 && "${ovn_encap_ip}" != "" ]]; then
-    OVN_ENCAP_IP=$(echo --encap-ip=${ovn_encap_ip})
+  ovn_encap_ip=$(ovs-vsctl --if-exists get Open_vSwitch . external_ids:ovn-encap-ip)
+  if [[ $? == 0 ]]; then
+    ovn_encap_ip=$(echo ${ovn_encap_ip} | tr -d '\"')
+    if [[ "${ovn_encap_ip}" != "" ]]; then
+      OVN_ENCAP_IP=$(echo --encap-ip=${ovn_encap_ip})
```

Comment:
Just do `OVN_ENCAP_IP="--encap-ip=${ovn_encap_ip}"`, no need for the echo.

Reply:
But actually, do we need to do this? even if the ovs-vsctl fails in the original code, ovn_encap_ip will be "" and we won't set OVN_ENCAP_IP to anything. Can you explain the error you were seeing in the original code? I'm probably just missing something.

Reply:
@dcbw this is just to err on the side of caution.  The fear was that if ovs-vsctl fails and writes something to sdout (instead of stderr) then we would run `tr` on that output and set ovn_encap_ip to something incorrect and we proceed.

Reply:
@girishmg ok